pages/map/[tablename].vue

```diff
@@ -57,12 +57,23 @@ if (data.value && !error.value) {
 }
 
 const { t } = useI18n();
+
+// Check if this view is publicly accessible
+const isPublic = computed(() => {
+  return isPublicView(data.value?.routeLevelPermission);
+});
+
```

Comment:
I had a comment about turning this into a utility module that each `[tablename].vue` page imports, to improve maintainability and so you can do

```javascript
import { useIsPublic } from "@/utils/permissions";

...

const isPublic = useIsPublic(data);
```

 I don't see that comment anymore and I also don't see a code change. If you disagree, can you briefly respond?